ppb/systems/renderer.py

```diff
@@ -107,6 +113,9 @@ def __del__(self):
 
 class Renderer(SdlSubSystem):
     _sdl_subsystems = SDL_INIT_VIDEO
+    last_opacity: int = 255
+    last_opacity_mode: str = 'blend'
+    last_color: Tuple[int, int, int] = (255, 255, 255)
```

Comment:
Are these doing anything? I don't see them referenced elsewhere?

Reply:
this was part of an attempt I made to only call the SDL `Set*` functions when these changed, but I didn't get it working right and dropped it. I will clean this up.

Reply:
Was just checking!